modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Select/Select.soy

```diff
@@ -184,31 +184,8 @@
 	{@param? multiple: bool}
 
 	<select class="form-control hide" id="{$name}" name="{$name}" {if $multiple}multiple size="{length($options)}"{/if}>
-		<option disabled {if length($displayValues) == 0}selected{/if} value=""></option>
-
-		{foreach $option in $options}
-			{call .select_hidden_options}
-				{param option: $option /}
-				{param values: $displayValues /}
-			{/call}
+		{foreach $currentValue in $displayValues}
+			<option selected="selected" value="{$currentValue}">{$currentValue}</option>
```

Comment:
Hey @linolaoj, should each option be selected? The previous logic was iterating over the `values` before take the decision to select or not the option.

Reply:
Hi @natocesarrego , displayValues has only the selected values, we need to iterate in case multiple choice is enabled.

Reply:
Good! Thanks for your explanation, @linolaoj. 😉